.vscode/settings.json

```diff
@@ -5,7 +5,8 @@
 	"editor.wordWrapColumn": 170,
 	"editor.formatOnSave": false,
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
+		"source.fixAll.eslint": true,
+		"source.fixAll.stylelint": true
```

Comment:
Why are you adding this?

Reply:
This will help you with lint ;)
If you install the [stylelinty](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) you will have your code styled each time you save the document.